Makefile

```diff
@@ -103,19 +108,19 @@ kolibri.apk.unsigned: p4a_android_distro src/kolibri needs-version
 # Makes dummy file
 .PHONY: build_docker
 build_docker: Dockerfile
-	docker build -t android_kolibri .
+	$(DOCKER) build -t android_kolibri .
 
 # Run the docker image.
 # TODO Would be better to just specify the file here?
 run_docker: build_docker
-	./scripts/rundocker.sh
+	env DOCKER="$(DOCKER)" ./scripts/rundocker.sh
```

Comment:
Equivalently, you can `export DOCKER` where it's set above like you would in a shell script. Otherwise this is basically the same as what I have locally to allow `DOCKER=podman`. I kind of like the `export` approach so you don't have to think about the places you want to pass it down, but there aren't many places it happens here so probably not a big deal.